modules/update-manifest-application.adoc

```diff
@@ -43,12 +43,16 @@ This is especially true for cluster Operators, which might perform their own upd
 While the additional update actions take place, these cluster Operators temporarily set their `Progressing` condition to `True`.
 ====
 
+// to do: potentially reword the note above to clarify that specific resources are being applied at one time, and not necessarily all the resources for that component.
+
 The CVO waits until all cluster Operators in the Runlevel meet the following conditions before it proceeds to the next Runlevel:
 
 * The cluster Operators have an `Available=True` condition.
 
 * The cluster Operators have a `Degraded=False` condition.
 
+// to do: potentially clarify that this condition is not applicable during installations, and also potentially add documentation (here or elsewhere) that explains how the CVO is constantly reconciling states whether or not an update is happening.
```

Comment:
Maybe I'd really add a "Cluster Version Operator"  section right to the start of "How cluster updates work." page. It would be fairly high-level and it would mention 1) that it is and operator, 2) it applies manifests from release images (also sounds like "Release Images" should be above the "Evaluation of..." one?) 3) how it does it during installation, over the course of normal cluster operation, and during the upgrade - and the upgrade mention sould be accompanied  with a link to the detailed procedure that we already have.